tox.ini

```diff
@@ -27,10 +27,10 @@ commands=
 
 [gh-actions]
 python =
-    2.7: py27
-    3.5: py35
-    3.6: py36
-    3.7: py37
     3.8: py38
```

Comment:
3.8 is now EOL as well: https://devguide.python.org/versions/

Reply:
I would love to see Python 3.8 kept at least for one release to ease migration path for us from 3.8 to 3.12. Thanks for considering!